openpype/hosts/maya/api/lib.py

```diff
@@ -2528,13 +2528,65 @@ def load_capture_preset(data=None):
             else:
                 temp_options2['ssaoEnable'] = False
 
+        if key == 'ssaoSamples':
+            temp_options2['ssaoSamples'] = preset[id][key]
+
+        if key == 'ssaoAmount':
+            temp_options2['ssaoAmount'] = preset[id][key]
+
+        if key == 'ssaoRadius':
+            temp_options2['ssaoRadius'] = preset[id][key]
+
+        if key == 'ssaoFilterRadius':
+            temp_options2['ssaoFilterRadius'] = preset[id][key]
+
         if key == 'alphaCut':
             temp_options2['transparencyAlgorithm'] = 5
             temp_options2['transparencyQuality'] = 1
 
         if key == 'headsUpDisplay':
             temp_options['headsUpDisplay'] = True
 
+        if key == 'hwFogEnable':
+            if preset[id][key] is True:
+                temp_options2['hwFogEnable'] = True
+
+            else:
+                temp_options2['hwFogEnable'] = False
```

Comment:
Is this because you need to be sure `preset[id][key]` has to be bool?

Because otherwise it could be much shorter
```suggestion
        if key == 'hwFogEnable':
            temp_options2['hwFogEnable'] = preset[id][key] or False
```

Reply:
Thank you @antirotor I'll commit the suggestion!